apprentice_teacher.rb

```diff
@@ -9,43 +13,6 @@ def initialize(options={})
     @phase = 3
   end
```

Comment:
Similarly, refactor this guy (initialize).  It's the same as the one in `Teacher` aside from one line!